src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5158,7 +5158,7 @@ def remove_github_actions(cmd, resource_group, name, repo, token=None, slot=None
         branch.replace('/', '-'), name.lower(), slot) if slot else "{}_{}.yml".format(
             branch.replace('/', '-'), name.lower())
     dir_path = "{}/{}".format('.github', 'workflows')
-    file_path = "/{}/{}".format(dir_path, file_name)
+    file_path = "{}/{}".format(dir_path, file_name)
```

Comment:
Also this path assumption will work only on Windows file system - shouldn't this be agnostic to the OS's file system?